src/hono.test.ts

```diff
@@ -2001,8 +2004,8 @@ describe('Context binding variables', () => {
 
   it('Should get binding variables with correct types', async () => {
     app.get('/', (c) => {
-      type verifyID = Expect<Equal<number, typeof c.env.USER_ID>>
-      type verifyName = Expect<Equal<string, typeof c.env.USER_NAME>>
+      expectTypeOf(c.env.USER_ID).toEqualTypeOf<number>()
```

Comment:
It does not pass, though it seems to be correct. What do you think to write like the following instead of that as a workaround:

```ts
 expectTypeOf(c.env).toEqualTypeOf<Bindings>()
```